xbmc/platform/android/jni/AudioFormat.cpp

```diff
@@ -76,6 +77,7 @@ void CJNIAudioFormat::PopulateStaticFields()
     if (sdk >= 5)
     {
       CJNIAudioFormat::CHANNEL_OUT_STEREO = get_static_field<int>(c, "CHANNEL_OUT_STEREO");
+      CJNIAudioFormat::CHANNEL_OUT_MONO = get_static_field<int>(c, "CHANNEL_OUT_MONO");
```

Comment:
above stereo so it looks pretty and matches the order define order. Yea, yea OCD is a bitch :)